website/src/models/user.js

```diff
@@ -390,6 +390,15 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
+    },
+    improvementCategories: {
```

Comment:
i had planned to use the user PUT route. But I can also create a new route for this if you think that is better. I just thought it would be a bit overkill.